client/src/test/scala/com.gu.contentapi.client/model/ContentApiQueryTest.scala

```diff
@@ -5,27 +5,27 @@ import org.scalatest.{Matchers, FlatSpec}
 class ContentApiQueryTest extends FlatSpec with Matchers  {
   "ItemQuery.toString" should "be excellent" in {
     ItemQuery("profile/robert-berry").showFields("all").toString shouldEqual
-      "ItemQuery(/profile/robert-berry?show-fields=all)"
+      "/profile/robert-berry?show-fields=all"
```

Comment:
It feels like `toString` isn't the method we should check here. We should be able to change how the query would look like when printed (typically for debugging) without having to change the tests.

In my opinion, we should be comparing the output of `.apply("")`.

Reply:
Thanks. Having done that, I'd suggest reverting the implementation of `toString` to include the class name. It used to make things a bit easier when debugging some issues.